### PR TITLE
[UI]Fix taskPriority propType validation error

### PR DESCRIPTION
**Pull request by katherine95**
#### What does this PR do?
This PR closes [Issue](https://github.com/taskcluster/taskcluster/issues/1406) concerning the propType validation console error for `taskPriority` at `/hooks/garbage/abc456`

#### Type of change:
- [x] Bug fix (non-breaking change which fixed an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update.

#### How Has This Been Tested?
- Clone this repository 
- Git checkout to `katherine95:Issue-1406`
- Run `yarn` then `yarn start`
- Navigate to hook view  `/hooks/garbage/abc456` and check the console
- You should see that there is no propType validation error on the console

#### Checklist:
- [x] Navigating to the hook view shows no error in the console.

#### Before
Screenshot 2019-10-03 at 18 21 25

#### After
Screenshot 2019-10-14 at 21 14 14

### CODE REVIEW

File: ui/src/utils/prop-types.js [hunk 1]
```diff
@@ -126,6 +126,7 @@ export const taskPriority = oneOf([
   'LOW',
   'VERY_LOW',
   'LOWEST',
+  'normal',
```
Comment (helfi92) [inline ui/src/utils/prop-types.js]: Ah, we are indeed missing the normal priority of a task. The convention in graphql is to capitalize constants then in web-server we can map the capital letters to their actual values.

**Comment by helfi92** [comment]
@katherine95 Did you want to finish the PR? :-)

**Comment by katherine95** [comment]
Yes @helfi92. Let me look at the feedback given. Thank for feedback.

File: ui/src/utils/prop-types.js (hunk 1 above)
Comment (katherine95) [reply ui/src/utils/prop-types.js]: So what I should do is capitalize on the `normal` constant?, It's not piking it when it's normal. So it seems it's picking data that is already from the db and the taskPriority is in small letters.

Comment (helfi92) [reply ui/src/utils/prop-types.js]: The code to make it work is on the diff that I posted on my review. Also, whenever we change things in `web-server`, we currently need to restart the process to see the changes. Does that help?

Comment (katherine95) [reply ui/src/utils/prop-types.js]: Yes that really helped. Thanks @helfi92

**Comment by katherine95** [comment]
@helfi92 please review the new changes.

**Comment by helfi92** [comment]
@all-contributors please add @katherine95 for code.

**Pull request merged**